backend/api/Services/MissionDefinitionService.cs

```diff
@@ -264,10 +264,6 @@ private IQueryable<MissionDefinition> GetMissionDefinitionsWithSubModels(
                 .ThenInclude(area => area!.Installation)
                 .Include(missionDefinition => missionDefinition.Source)
                 .Include(missionDefinition => missionDefinition.LastSuccessfulRun)
-                .ThenInclude(missionRun => missionRun != null ? missionRun.Robot : null)
-                .Include(missionDefinition => missionDefinition.LastSuccessfulRun)
-                .ThenInclude(missionRun => missionRun != null ? missionRun.Tasks : null)!
-                .ThenInclude(missionTask => missionTask.Inspection)
```

Comment:
If removing the task include, the function ScheduleMissionRunFromMissionDefinitionLastSuccessfullRun needs to be updated so it doesn't fetch tasks like this anymore "missionDefinition.LastSuccessfulRun.Tasks"